src/components/authorise/authorise-get-controller.ts

```diff
@@ -20,6 +20,17 @@ export const authoriseGetController = (req: Request, res: Response): void => {
       config
     );
 
+    if (config.getAuthoriseErrors().includes("ACCESS_DENIED")) {
+      logger.warn("Client configured to return access_denied error response");
+      throw new AuthoriseRequestError({
+        errorCode: "access_denied",
+        errorDescription: "access_denied",
```

Comment:
entirely my fault for bad spec haha, but could we use the default description from the oauth2errorclass
```suggestion
        errorDescription: "Access denied by resource owner or authorization server",
```

Reply:
Ah yeah probably should've looked at this, it's now fixed!